CRM/Eventbrite/Form/Manage/Event.php

```diff
@@ -121,7 +121,7 @@ public function postProcess() {
       }
       $result = _eventbrite_civicrmapi('EventbriteLink', 'create', $apiParams);
     }
-    CRM_Core_Session::setStatus(ts('Settings have been saved.'), ts('Saved'), 'success');
+    CRM_Core_Session::setStatus(ts('Settings have been saved.'), E::ts('Saved'), 'success');
```

Comment:
@lucky091588 Looks like there's one more `ts` call here that needs the same treatment.